src/main/resources/templates/fragments/cardHeader.html

```diff
@@ -1,7 +1,4 @@
-<!DOCTYPE html>
-<html th:lang="${#locale.language}" xmlns:th="http://www.thymeleaf.org">
-<body>
-<header
+<div
```

Comment:
Wasn't sure if this was better off as a `<th:block>` or `<div>`.  Thoughts @coltborg?

Reply:
Here's my thought process on if to use a `<th:block>`:
* Do I want to an element rendered to the page?
  * If yes:
    * What is the most semantic element I can use? I'll try to use that.
    * If it's for display purposes or something technical that doesn't have useful information for a client, I'll pick a `<div>`.
  * If no:
    * I'll use `<th:block>`.

Reply:
For this one, I update the element and used `<header>` instead of `<div>`.